### PR TITLE
Release Preperation 

### PR DESCRIPTION
**Pull request by marcbonnici**

### CODE REVIEW

File: doc/source/changes.rst [hunk 1]
```diff
@@ -2,6 +2,95 @@
...
+             will be support Python 3.5+ only.
+
+New Features:
+==============
+
+Framework:
+----------
+    - A maximum and minimum version of an APK can be specified for ``ApkWorkloads``
+      instead of required a specific version.
+    - ``Artifacts`` now support the addition of directories.
```
Comment (setrofim) [inline doc/source/changes.rst]: I would rephrase this as 

```
-  Directories can now be added as ``Artifacts``.
```

File: doc/source/changes.rst [hunk 2]
```diff
@@ -2,6 +2,95 @@
...
+Other:
+------
+    - Added ``requirements.txt`` as a reference for known working package versions.
+
+Fixes/Improvements
+==================
+
+Framework:
+----------
+    - ``JobOuputs`` now have an ``augmentation`` attribute to allow for easy listing
```
Comment (setrofim) [inline doc/source/changes.rst]: Should the attribute name be pluralised?

Comment (marcbonnici) [reply doc/source/changes.rst]: Updated.

File: doc/source/changes.rst (hunk 1 above)
Comment (marcbonnici) [reply doc/source/changes.rst]: Updated.
